src/BloomBrowserUI/publish/ReaderPublish/ReaderPublishScreen.tsx

```diff
@@ -128,6 +128,11 @@ const ReaderPublishScreenInternal: React.FunctionComponent<{
                 case "stopped":
                     setClosePending(true);
                     break;
+                case "UpdatingPreview":
+                    setProgressState(ProgressState.Working);
```

Comment:
Please reorder this to make it more parallel with the other cases.

Reply:
ok fixed